generator/transformers/common_producer.py

```diff
@@ -337,7 +337,7 @@ def extract_param(self, param: Param, item_name: str):
                 'mandatory': param.is_mandatory,
                 'deprecated': json.loads(param.deprecated.lower()) if param.deprecated else False,
                 'modifier': 'strong'}
-        if isinstance(param.param_type, (Integer, Float, String, Array)):
+        if isinstance(param.param_type, (Integer, Float, String, Array, Boolean)):
```

Comment:
What is the purpose of this change? The only thing it seems to do is add a `{}` to the `shifted` parameter, but since it has no data it doesn't seem to be helpful without additional work. It also doesn't seem to be related to this change, so I think it can be reverted.